### DISCUSSION TITLE
Help me get GB

### DISCUSSION THREAD

**Discussion by Sriansh-raj** [drknzz/GitHub-Achievements#118] [Q&A] [answered]
Hey there !!

**Comment by luizbizzio** [comment, marked as answer]
Of course I will help you :)

How to get:
Answered a discussion
(got an accepted answer)

**Reply by Sriansh-raj** [reply]
> Of course I will help you :)
> 
> How to get: Answered a discussion (got an accepted answer)

Thank you <3 😄
